platform/wasm/netlify.sh

```diff
@@ -4,11 +4,29 @@ set -eux -o pipefail
 
 PROJECT=debugtox
 
-tar -C /opt -zxf <(curl -L https://github.com/TokTok/dockerfiles/releases/download/nightly/qtox-wasm-buildhome.tar.gz)
+if [ ! -f /opt/buildhome/bin/ccache ]; then
+  mkdir -p /opt/buildhome/bin
+  tar -C /opt/buildhome/bin --strip-components=1 -Jxf \
+    <(curl -L https://github.com/ccache/ccache/releases/download/v4.10.2/ccache-4.10.2-linux-x86_64.tar.xz) \
+    ccache-4.10.2-linux-x86_64/ccache
+fi
+if [ ! -d /opt/buildhome/emsdk ]; then
+  tar -C /opt -zxf \
+    <(curl -L https://github.com/TokTok/dockerfiles/releases/download/nightly/qtox-wasm-buildhome.tar.gz)
+fi
 
 . '/opt/buildhome/emsdk/emsdk_env.sh'
+export PATH="/opt/buildhome/bin:$PATH"
 export PKG_CONFIG_PATH='/opt/buildhome/lib/pkgconfig'
 
+# Limit ccache to 100MB storage.
+ccache -M 100M
+ccache --show-stats
+ccache --zero-stats
+
+export CC='ccache emcc'
+export CXX='ccache em++'
```

Comment:
This kind of stuff is usually done by passing `-DCMAKE_C_COMPILER_LAUNCHER=sccache` to cmake, but I don't know if emcmake does anything that breaks that.